resources/fetchers/logging_fetcher.go

```diff
@@ -68,9 +68,11 @@ func (f LoggingFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetad
 		f.log.Errorf("failed to describe config recorders: %v", err)
 	}
 
-	f.resourceCh <- fetching.ResourceInfo{
-		Resource:      ConfigResource{configs: configs, identity: f.cloudIdentity},
-		CycleMetadata: cMetadata,
+	if len(configs) > 0 {
```

Comment:
i'm not sure, but maybe if the are no configs at all, the rule should fail?

Reply:
how can it happen? the config is either enabled or disabled, no?

Reply:
`configs` can be empty when `MultiRegionFetch` fails to fetch and returns an empty `[]awslib.AwsResource` and an error. we log the error but shouldn't push to the resource channel. in second thought tho this shouldn't fail the rule as it's not operating on any resources.

Reply:
replaced the `count` check with returning early when we get an error